tools/clean-emoji-cache

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+from __future__ import absolute_import
+from __future__ import print_function
+import os
+import subprocess
+import sys
+import hashlib
+
+ZULIP_PATH = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+EMOJI_CACHE_PATH = "/srv/zulip-emoji-cache"
+if "--travis" in sys.argv:
+    EMOJI_CACHE_PATH = os.path.join(os.environ["HOME"], "zulip-emoji-cache")
+
+sha = hashlib.sha1()
+filenames = ['NotoColorEmoji.ttf', 'emoji_map.json', 'AndroidEmoji.ttf',
+             'build_emoji', 'emoji_setup_utils.py']
+for filename in filenames:
+    file_path = os.path.join(ZULIP_PATH, 'tools', 'setup', 'emoji', filename)
+    with open(file_path, 'rb') as reader:
+        sha.update(reader.read())
+success_stamp = os.path.join(EMOJI_CACHE_PATH, sha.hexdigest(),
+                             'emoji', '.success-stamp')
+
```

Comment:
I copied up some stuff here (Line 14-22) from `tools/setup/build/emoji_build` since there seemed no obvious way to import the function and to do it we needed to convert the module into a package and also put in `.py` in its name.
What are thoughts on doing that?
I believe the current way shouldn't hurt much though.